src/nunavut/lang/properties.ini

```diff
@@ -277,3 +277,15 @@ named_values =
     true = true
     false = false
     null = null
+
+[nunavut.lang.docgen]
+extension = .html
+has_standard_namespace_files = True
+namespace_is_composite_type = False
+namespace_file_stem = Namespace
+named_values =
+    true = true
+    false = false
+    null = null
+options =
+    omit_serialization_support = true
```

Comment:
I'm not a fan of this. First off, command line overrides should dominate so if you specify "always" you should override this option (see above). Instead we should make the generator fail gracefully if the language simply doesn't have any serialization support.